VAMobile/src/screens/BenefitsScreen/ClaimsScreen/AppealDetailsScreen/AppealStatus/AppealCurrentStatus/AppealCurrentStatus.tsx

```diff
@@ -405,10 +405,12 @@ function AppealCurrentStatus({ status, aoj, appealType, docketName, programArea
               <TextView variant="MobileBody">{details[1]}</TextView>
               <TextView variant="MobileBodyLink">{details[2]}</TextView>
               <TextView variant="MobileBody">{details[3]}</TextView>
+            </TextView>
+            <TextView>
               <TextView variant="MobileBodyLink" onPress={(): void => launchExternalLink(LINK_URL_DECISION_REVIEWS)}>
```

Comment:
Can the links in this section use LinkWithAnalytics instead? Optional since it may be out of scope for the ticket

Reply:
I think this is out of scope and likely needs to be addressed all together across the whole file in a different ticket